samples/display/st7789v/src/main.c

```diff
@@ -121,6 +121,7 @@ void main(void)
 			*(buf + idx + 1) = (color_rgb >> 0) & 0xFFU;
 		}
 #else
+		int color = cnt % 4;
```

Comment:
is this a stray change? it doesn't seem related to the commit message `samples: display: Use ST7789V generic shield`

Reply:
Looks like it, but it is a bug fix for the sample in case RGB565 is not defined

Reply:
Not nice but this sample (and any other controller type specific) should be removed after we merged #20580